contrib/touched-files-check/src/main.rs

```diff
@@ -7,7 +7,14 @@ fn check_attestations(atts: Vec<&str>, mut keys: HashSet<&str>) -> Result<(), St
         let builder_key = format!("builder-keys/{builder}.gpg");
         keys.remove(&builder_key as &str);
         let builder_key_file = std::fs::File::open(&builder_key)
-            .map_err(|e| format!("Builder key not found for attestation. Attestation: '{att}', Key: '{builder_key}', Error: '{e}'"))?;
+            .map_err(|e| format!("Builder key not found for attestation. Attestation: '{att}', Key: '{builder_key}', Error: '{e}'.\nHelp: Run 'gpg --export --armor {builder} > {builder_key} && git add {builder_key}'"))?;
```

Comment:
Looks like the `\n` remains escaped in the CI log. Maybe a good first issue if someone wants to fix this in a follow-up?